frontend/src/scenes/trends/PersonsModal.tsx

```diff
@@ -148,7 +159,8 @@ export function PersonsModal({ visible, view, filters, onSaveCohort }: PersonsMo
                             <span>
                                 This list contains{' '}
                                 <b>
-                                    {people.count} unique {pluralize(people.count, 'user', undefined, false)}
+                                    {people.count} unique{' '}
+                                    {pluralize(people.count, _isGroupType ? 'group' : 'user', undefined, false)}
```

Comment:
Blocker: We don't use the word group about groups anywhere in the app.

It's always `organization(s)`, `project(s)`, etc. Check out the existing logic on this.